src/main/java/org/openimmunizationsoftware/cdsi/core/logic/EvaluateForPreferableVaccine.java

```diff
@@ -39,49 +39,37 @@ public EvaluateForPreferableVaccine(DataModel dataModel) {
           new ConditionAttribute<Date>("Vaccine dose administered", "Date Administered");
 
       AntigenAdministeredRecord aar = dataModel.getAntigenAdministeredRecord();
-      logicTable.caPreferableVaccineTradeName = new ConditionAttribute<String>(
-          "Supporting data (Preferable Vaccine)", "Preferable Vaccine Trade Name ");
+      logicTable.caVolume = new ConditionAttribute<String>(
+          "Vaccine dose administered", "Preferable VaccicaPreferableVaccineTypene Volume ");
       logicTable.caTradeName =
           new ConditionAttribute<String>("Vaccine dose administered", "Trade Name ");
+      logicTable.caPreferableVaccineElements = new ConditionAttribute<String>(
+          "Supporting Data", "Preferable Vaccine elements");
       logicTable.caVaccineTypeBeginAgeDate = new ConditionAttribute<Date>(
           "Vaccine Type Begin Age Date", "Calculated date (CALCDTPREF-1)");
       logicTable.caVaccineTypeEndAgeDate = new ConditionAttribute<Date>("Vaccine Type End Age Date",
           "Calculated date (CALCDTPREF-2)");
-      logicTable.caPreferableVaccineVolume = new ConditionAttribute<String>(
-          "Preferable Vaccine Volume ", "Supporting data (Preferable Vaccine)");
-      logicTable.caPreferableVaccineType = new ConditionAttribute<String>(
-          "Preferable Vaccine Type ", "Supporting data (Preferable Vaccine)");
-      logicTable.caVolume = aar.getVolume();
-      logicTable.caTradeName.setInitialValue(aar.getAntigen().getName());
-      // caPreferableVaccineTradeName.setInitialValue(
-      // dataModel.getAntigenSeriesList().get(0).getSeriesDoseList().get(0).getAllowableVaccineList().get(0).getTradeName())
-      // ;
+
+      logicTable.caVolume.setInitialValue(aar.getVolume());
       logicTable.caDateAdministered.setInitialValue(aar.getDateAdministered());
 
       logicTable.caVaccineTypeBeginAgeDate.setAssumedValue(PAST);
       logicTable.caVaccineTypeEndAgeDate.setAssumedValue(FUTURE);
-      logicTable.caPreferableVaccineTradeName.setAssumedValue(aar.getAntigen().getName());
-      logicTable.caPreferableVaccineVolume.setAssumedValue(aar.getVolume());
+      
+      logicTable.caPreferableVaccineVolume.setAssumedValue(pi.getVolume());
```

Comment:
So it doesn't look like " caPreferableVaccineVolume" is still a field in the new table.